Gems/PhysX/Code/Editor/Source/Components/Conversion/CollidersPrefabConversion.cpp

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <AzCore/Asset/AssetManager.h>
+#include <AzCore/Console/IConsole.h>
+
+#include <AzFramework/API/ApplicationAPI.h>
+
+#include <EditorColliderComponent.h>
+#include <EditorShapeColliderComponent.h>
+#include <EditorRigidBodyComponent.h>
+#include <EditorStaticRigidBodyComponent.h>
+
+#include <Editor/Source/Components/Conversion/PrefabConversionUtils.h>
+
+namespace PhysX
+{
+    void FixPrefabsWithColliderComponents(const AZ::ConsoleCommandContainer& commandArgs);
+
+    AZ_CONSOLEFREEFUNC(
+        "ed_physxFixPrefabsWithColliderComponents",
+        FixPrefabsWithColliderComponents,
+        AZ::ConsoleFunctorFlags::Null,
+        "Finds entities with collider components and no rigid bodies and fixes them by adding a static rigid body component.");
+
+    bool AddStaticRigidBodyToPrefabEntity(
+        Utils::PrefabInfo& prefabInfo,
+        AzToolsFramework::Prefab::PrefabDomValue& entityPrefab)
```

Comment:
nit: (const)
```suggestion
        const Utils::PrefabInfo& prefabInfo,
        const AzToolsFramework::Prefab::PrefabDomValue& entityPrefab)
```

Reply:
It's not possible to pass `prefabInfo` because it needs to use the non-const version of `prefabInfo.m_template->GetPrefabDom()` (it needs to use the allocator)